android/titanium/src/java/org/appcelerator/titanium/view/TiUIView.java

```diff
@@ -1427,14 +1427,18 @@ private void initializeBorder(KrollDict d, Integer bgColor)
 					borderView.setColor(TiConvert.toColor(d, TiC.PROPERTY_BORDER_COLOR));
 				}
 
+				//Have a default border width of 1
+				Object borderWidth = "1";
```

Comment:
I've confirmed that Titanium for iOS defaults the `borderWidth` to "1dp" (Apple's framework takes UI measurements in points/dips by default). This can be seen in the `[TiUIView setBorderColor]` method here...
https://github.com/appcelerator/titanium_mobile/blob/master/iphone/Classes/TiUIView.m#L483

So, I recommend that you change "1" to "1dp" here to match iOS' behavior.

Reply:
@jquick-axway I set it as a String with value '1' to use the default measure unit that is set in tiapp.xml. I think iOS does the same here:
https://github.com/appcelerator/titanium_mobile/blob/master/iphone/Classes/TiDimension.m#L103